site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Segmented Controls
+description: Segmented controls component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - segmented-controls
+---
+
+# Segmented Control
+
+Segmented control component consists of navigating to the previous or next segmented control item by using the arrow keys or clicking on a segmented control. The result will change the content to display the current selected item. To do this the `[data-segmented-control="1"]` and `[data-segmented-control-item="1"]` must have matching numbers as these act as the index for navigation.
```

Comment:
> "To do this the `[data-segmented-control="1"]` and `[data-segmented-control-item="1"]` must have matching numbers as these act as the index for navigation."

This bit's no longer accurate, so these docs should be updated. Currently, a `[data-segmented-control]` attribute is required, since it's the selector we're using to get the list of tabs, but only in a boolean present/absent sense - it doesn't need any value at all, let alone any specific index.

Reply:
This should be gone now.